### PR TITLE
🌐 Add Ukrainian translation for `docs/uk/docs/tutorial/cookie-params.md`

### PR DESCRIPTION
**Pull request by rostik1410**
related: https://github.com/tiangolo/fastapi/issues/1748

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 96bf47436c0315dc90047a607be84ec71c2458bb at: https://d3e7de37.fastapitiangolo.pages.dev

**Review by mpmeleshko — APPROVED** [review]

**Review by Vlad0395 — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 382f86367974207e2b8cb4473de5783e00efc5b4 at: https://96909110.fastapitiangolo.pages.dev

**Review by bisibuka — APPROVED** [review]
LGTM

**Comment by tiangolo** [comment]
Great, thanks @rostik1410! 🍰 

And thanks for the reviews @bisibuka, @mpmeleshko, @Vlad0395 🙇 🎉

File: docs/uk/docs/tutorial/cookie-params.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    ```Python hl_lines="9"
+    {!> ../../../docs_src/cookie_params/tutorial001.py!}
+    ```
+
+!!! note "Технічні Деталі"
+    `Cookie` це "сестра" класів `Path` і `Query`. Вони наслідуються від одного батьківського класу `Param`.
+    Але пам'ятайте, що коли ви імпортуєте `Query`, `Path`, `Cookie` та інше з `fastapi`, це фактично функції, що повертають спеціальні класи.
+
+!!! info
+    Для визначення cookies ви маєте використовувати `Cookie`, тому що в іншому випадку параметри будуть інтерпритовані, як параметри запиту.
```
Comment (ArtemKhymenko) [inline docs/uk/docs/tutorial/cookie-params.md:92]: інтерпретовані instead of інтерпритовані

**Pull request merged**